coffee_httpd/src/httpd/server.rs

```diff
@@ -73,6 +75,48 @@ async fn coffee_help(
     Ok(body)
 }
 
+#[api_v2_operation]
+#[post("/install")]
+async fn coffee_install(
+    data: web::Data<AppState>,
+    body: Json<HashMap<String, String>>,
```

Comment:
```suggestion
    body: Json<InstallBody>,
```

I guess you can do something similar and avoid to bring the hash map around

Reply:
Yes. I fixed this also for the previously implemented endpoints that accept a request body.
Now for testing, we can 
```
curl -X POST \
  http://localhost:8080/remote/rm \
  -H 'accept: application/json' \
  -H 'Content-Type: application/json' \
  -d '{
    "repository_name": "Folgore"
}'
```